content/webapp/pages/books.tsx

```diff
@@ -0,0 +1,81 @@
+import type { GetServerSideProps } from 'next';
+import { getBooks } from '@weco/common/services/prismic/books';
+import PageLayout from '@weco/common/views/components/PageLayout/PageLayout';
+import LayoutPaginatedResults from '@weco/common/views/components/LayoutPaginatedResults/LayoutPaginatedResults';
+import { convertImageUri } from '@weco/common/utils/convert-image-uri';
+import { Book } from '@weco/common/model/books';
+import { PaginatedResults } from '@weco/common/services/prismic/types';
+import SpacingSection from '@weco/common/views/components/SpacingSection/SpacingSection';
+import {
+  getGlobalContextData,
+  WithGlobalContextData,
+} from '@weco/common/views/components/GlobalContextProvider/GlobalContextProvider';
+import { AppErrorProps } from '@weco/common/views/pages/_app';
+import { removeUndefinedProps } from '@weco/common/utils/json';
+import { FC } from 'react';
+
+type Props = {
+  books: PaginatedResults<Book>;
+} & WithGlobalContextData;
+
+const pageDescription =
+  'We publish thought-provoking books exploring health and human experiences.';
```

Comment:
It's not to address in this PR, but should strings like this live in a single file?

As I understand it, most of the text on the website is being pulled from Prismic. If we do have to hard-code strings, maybe we should keep them together so they can be reviewed or inspected in one place by the Editorial team?

Reply:
Nice point. This is super legacy code, as most SEO should be controlled in Prismic. I'd rather be doing the work to do that rather than facilitate this 4year old hack.